exec/java-exec/src/main/java/org/apache/drill/exec/store/log/README.md

```diff
@@ -11,26 +18,50 @@ If you wanted to analyze log files such as the MySQL log sample shown below usin
 070917 16:29:01      21 Query       select * from location
 070917 16:29:12      21 Query       select * from location where id = 1 LIMIT 1
 ```
-This plugin will allow you to configure Drill to directly query logfiles of any configuration.
+
+Using this plugin, you can configure Drill to directly query log files of
+any configuration.
 
 ## Configuration Options
-* **`type`**:  This tells Drill which extension to use.  In this case, it must be `logRegex`.  This field is mandatory.
-* **`regex`**:  This is the regular expression which defines how the log file lines will be split.  You must enclose the parts of the regex in grouping parentheses that you wish to extract.  Note that this plugin uses Java regular expressions and requires that shortcuts such as `\d` have an additional slash:  ie `\\d`.  This field is mandatory.
-* **`extension`**:  This option tells Drill which file extensions should be mapped to this configuration.  Note that you can have multiple configurations of this plugin to allow you to query various log files.  This field is mandatory.
-* **`maxErrors`**:  Log files can be inconsistent and messy.  The `maxErrors` variable allows you to set how many errors the reader will ignore before halting execution and throwing an error.  Defaults to 10.
-* **`schema`**:  The `schema` field is where you define the structure of the log file.  This section is optional.  If you do not define a schema, all fields will be assigned a column name of `field_n` where `n` is the index of the field. The undefined fields will be assigned a default data type of `VARCHAR`.
+
+* **`type`**:  This tells Drill which extension to use.  In this case, it must
+be `logRegex`.  This field is mandatory.
+* **`regex`**:  This is the regular expression which defines how the log file
+lines will be split.  You must enclose the parts of the regex in grouping
```

Comment:
Looks like everywhere in the doc there are two spaces before sentences instead of one. Could you please check and fix?